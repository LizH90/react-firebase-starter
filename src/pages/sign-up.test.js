import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import SignUp from './sign-up'

test('The sign up page renders correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <SignUp />
    </MemoryRouter>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
