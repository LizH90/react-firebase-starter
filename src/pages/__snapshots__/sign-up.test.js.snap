// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The sign up page renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
      Sign Up
    </h1>
    <div
      className="App"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          onChange={[Function]}
          placeholder="Email"
          value=""
        />
        <input
          onChange={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
        <button
          type="submit"
        >
          Sign Up
        </button>
      </form>
    </div>
    <a
      href="/sign-in"
      onClick={[Function]}
    >
      Back to Sign In
    </a>
  </header>
</div>
`;
